### DISCUSSION TITLE
Please add "play next file automatically" Button to Playlist

### DISCUSSION THREAD

**Discussion by viper73** [smplayer-dev/smplayer#239] [Ideas]
Hello,

i use smplayer a long time now and it is the best player i ever had!

I am just missing one litte thing:
In the "Playlist" i am missing an button "play next file automatically" to switch this on and off (alternatively an button for the main toolbar).

I switch this very often fron on to off and have to go to the preferences every time.
Maybe it is possible to add such an feauture in an future update. Would be great!

Greets, Viper73

**Comment by Don1776** [comment]
I also would like a button for "play next file automatically" for the Playlist.
